firestore/tests/unit/test__helpers.py

```diff
@@ -1333,7 +1321,7 @@ def test_field_updates(self):
             ('f.g', SERVER_TIMESTAMP),
         ))
         field_paths, actual_data = self._call_fut(data)
-        self.assertEqual(field_paths, ['c.d.e', 'f.g'])
+        self.assertEqual(field_paths, ['`c.d`.e', 'f.g'])
```

Comment:
Those back-ticks are actually expected here?  Do we have docs which define that?

Reply:
It does seem a little strange to me also, but I think it might have to do with how using 'a.b' is actually discouraged because you can't tell whether it is a nest or a field name.  I'm trying to work on some issues relating to this.  Adding a do not merge tag.